releasenotes/notes/mps_print_bond_dimension-6dc6f6cf0365c2ae.yaml

```diff
@@ -0,0 +1,10 @@
+---
+features:
+  - |
+    If the environment variable `MPS_OUTPUT_DATA` is set to 1, then MPS
```

Comment:
Is the environment variable still used? I think this should say how to use this feature by using the `set_options(MPS_log_data=True)` for the AerSimulator and QasmSimulator backend classes.

Reply:
Indeed I removed the environment variable. You are right that it looks better simply as an option.